server/src/main/java/com/google/musicanalysis/site/VideoGenreCount.java

```diff
@@ -0,0 +1,12 @@
+package com.google.musicanalysis.site;
+
+/** contains number of youtube videos under a music genre */
+public class VideoGenreCount {
+  public final String genre;
+  public int count;
```

Comment:
also ... is it fine to have no `final/static` here for the `count` var? Otherwise can't increment the count.